updateengine/client.go

```diff
@@ -63,7 +63,7 @@ func New() (c *Client, err error) {
 		return nil, err
 	}
 
-	c.object = c.conn.Object("com.coreos.update1", dbus.ObjectPath(dbusPath))
+	c.object = c.conn.Object("com.coreos.update1", dbus.ObjectPath(dbusPath)).(*dbus.Object)
```

Comment:
Checked on my machine and this seems to be the line breaking the build - will edit tomorrow.

Reply:
If you think this related to different go toolchain version, feel free to add 1.9.x and 1.10.x to travis. Please note that locksmith relies on an old vendored copy of go-dbus for this, so double-check your environment and dependency resolution first.

Reply:
If you think this related to different go toolchain versions, feel free to add 1.9.x and 1.10.x to travis. Please note that locksmith relies on an old vendored copy of go-dbus for this, so double-check your environment and dependency resolution first.